npm/webpack-dev-server/tsconfig.json

```diff
@@ -24,7 +24,7 @@
     // "isolatedModules": true,               /* Transpile each file as a separate module (similar to 'ts.transpileModule'). */
 
     /* Strict Type-Checking Options */
-    "strict": false /* Enable all strict type-checking options. */,
+    "strict": true /* Enable all strict type-checking options. */,
```

Comment:
isn't typescript type checking strict by default?

Reply:
Nope

![Screen Shot 2021-05-14 at 10 01 42 AM](https://user-images.githubusercontent.com/238679/118304382-76250400-b49b-11eb-9276-cfd3efbd7eff.png)

Reply:
This is odd indeed...
Thank you for checking

Reply:
If you create a new `tsconfig.json` with `yarn tsc --init` is sets `"strict": true`.